pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -1,34 +1,26 @@
-from pythonforandroid.recipe import CompiledComponentsPythonRecipe, Recipe
-from pythonforandroid.logger import shprint, info
-from pythonforandroid.util import current_directory
-from multiprocessing import cpu_count
+from pythonforandroid.recipe import Recipe, MesonRecipe
 from os.path import join
-import glob
-import sh
 import shutil
 
 
-class NumpyRecipe(CompiledComponentsPythonRecipe):
+class NumpyRecipe(MesonRecipe):
+    version = 'v2.0.0rc1'
```

Comment:
Why do we need to target a `rc` version?
They are changing how things get built?

Do you know if they have an ETA for the final release?

Reply:
I did because of https://github.com/pandas-dev/pandas/blob/eadc1294b13bdc661283fcdf3ba54a50f31feeab/pyproject.toml#L9

Reply:
As this strict requirement belongs to the build_system, shouldn't we be fine to keep `numpy<2.0.0` ?

Reply:
(I'm not worried about rc, that would be super-fine if it was a feature release, but since is a major one, and I have no idea what `numpy` changed between 1.x.x and 2.x.x, I'm worried about breaking things for our users)

Reply:
If I set lower than `2.0.0` then compilation will fail because we are passing numpy includes as:

```python
        env['NUMPY_INCLUDES'] = join(
            self.ctx.get_python_install_dir(arch.arch), "numpy/_core/include",
        )
```

I am afraid if something bad will happen if left pandas to use numpy (build isolation one). But it's worth a try.

Reply:
Let's try, otherwise: https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml

`pandas==2.2.1` does not seem to be affected.